src/test/ij.c

```diff
@@ -119,6 +115,8 @@ hypre_int
 main( hypre_int argc,
       char *argv[] )
 {
+   MPI_Comm            comm = hypre_MPI_COMM_WORLD;
```

Comment:
This change is fine, but I don't understand why it's needed.  When will the communicator in our drivers ever be anything other than MPI_COMM_WORLD?

Reply:
Hi Rob, I agree. Let me know if you prefer to revert this

Reply:
Either way is fine with me.  I was mostly wondering if you had some other reason for making the change.  Thanks!